app/validators/special_character_validator.rb

```diff
@@ -0,0 +1,25 @@
+class SpecialCharacterValidator < ActiveModel::EachValidator
+
+  def initialize(**kwargs)
```

Comment:
This initialize method is just calling super ?  That should mean it can be removed